sdk/keyvault/azure-security-keyvault-keys/test/mocked_transport_adapter_test.hpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief The base class to construct and init a Key Vault client.
+ *
+ */
+
+#include <gtest/gtest.h>
+
+#include <azure/core.hpp>
+#include <azure/keyvault/key_vault.hpp>
+
+#include <cstdio>
+
+namespace Azure { namespace Security { namespace KeyVault { namespace Keys { namespace Test {
```

Comment:
It would be better if every "Test" was under the same namespace.
```suggestion
namespace Azure { namespace Test { namespace Security { namespace KeyVault { namespace Keys { 
```

Reply:
We do Azure::Core::Test and Azure::Storage::Test... 
This would be a more general change than this PR can do